src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkResourcesKeywordArgsInspection.kt

```diff
@@ -20,11 +23,48 @@ class SmkResourcesKeywordArgsInspection : SnakemakeInspection() {
 
             st.argumentList?.arguments?.forEach {
                 if (it !is PyKeywordArgument) {
-                    registerProblem(it, SnakemakeBundle.message("INSP.NAME.resources.unnamed.args"))
+                    val action = MoveCaretAndInsertEqualsSignAction(SmartPointerManager.createPointer(it))
+                    registerProblem(
+                            it,
+                            SnakemakeBundle.message("INSP.NAME.resources.unnamed.args"),
+                            ProblemHighlightType.ERROR,
+                            action,
+                            action
+                    )
                 }
             }
         }
     }
 
     override fun getDisplayName(): String = SnakemakeBundle.message("INSP.NAME.resources.unnamed.args")
+
+    private class MoveCaretAndInsertEqualsSignAction(
+            private val elementPointer: SmartPsiElementPointer<PsiElement>
+    ) : HintAction, LocalQuickFixAndIntentionActionOnPsiElement(elementPointer.element) {
+        override fun startInWriteAction() = true
+
+        override fun getFamilyName(): String = SnakemakeBundle.message("INSP.INTN.name.resource.family")
+
+        override fun showHint(editor: Editor) = true
+
+        override fun getText(): String = SnakemakeBundle.message("INSP.INTN.name.resource.text")
+
+        override fun invoke(
+                project: Project,
+                file: PsiFile,
+                editor: Editor?,
+                startElement: PsiElement,
+                endElement: PsiElement
+        ) {
+            if (!FileModificationService.getInstance().prepareFileForWrite(file)) {
+                return
+            }
+
+            val offset = elementPointer.element?.textOffset ?: return
+            editor?.caretModel?.moveToOffset(offset)
+            if (editor?.document?.isWritable == true) {
+                editor.document.insertString(offset, "=")
```

Comment:
Before
`resources: aaa = "aa", "bb"`
After
`resources: aaa = "aa", <caret>="bb"`

According to my current code style settings should be `resources: aaa = "aa", <caret> = "bb"`, please take in cosideration user code style settings (e.g. `Settings|Editor|Code Style|Python|Spaces|Around Operators|Around = in keyword argument`)

Reply:
done